pkg/snowflake/task.go

```diff
@@ -233,6 +244,16 @@ func (tb *TaskBuilder) RemoveComment() string {
 	return fmt.Sprintf(`ALTER TASK %v UNSET COMMENT`, tb.QualifiedName())
 }
 
+// ChangeAllowOverlappingExecution returns the sql that will change the ALLOW_OVERLAPPING_EXECUTION for the task.
+func (tb *TaskBuilder) ChangeAllowOverlappingExecution(flag bool) string {
```

Comment:
this function signature should probably be changed to `SetAllowOverlappingExecution(overlappingExecution bool) string {}`